### PR TITLE
Relates-to: #3155 converting duplicated code to fixture

### PR DESCRIPTION
**Pull request by sanders41**
Relates-to: #3155 

Converting duplicated code to a fixture.

### CODE REVIEW

**Comment by abn** [comment]
@sanders41 thank you for the contribution. Would be great if you can move the other packages (cleo, pendulum) in this file into an `autouse` fixture. All of these can in in a fixture like this:

```
@fixture(autouse=true)
def __add_packages(repo):
    ...
```

**Comment by sanders41** [comment]
@abn I made the update. If I misunderstood what you wanted just let me know and I can update further.

File: tests/console/commands/debug/test_resolve.py [hunk 1]
```diff
@@ -9,14 +9,20 @@ def tester(command_tester_factory):
     return command_tester_factory("debug resolve")
 
 
-def test_debug_resolve_gives_resolution_results(tester, repo):
+@pytest.fixture(autouse=True)
+def __add_packages(repo):
     cachy2 = get_package("cachy", "0.2.0")
     cachy2.add_dependency(Factory.create_dependency("msgpack-python", ">=0.5 <0.6"))
```
Comment (abn) [inline tests/console/commands/debug/test_resolve.py:15]:
```suggestion
    cachy020 = get_package("cachy", "0.2.0")
    cachy020.add_dependency(Factory.create_dependency("msgpack-python", ">=0.5 <0.6"))
```

While we are doing this, might has well name this right. :)

File: tests/console/commands/debug/test_resolve.py [hunk 2]
```diff
@@ -9,14 +9,20 @@ def tester(command_tester_factory):
...
 
     repo.add_package(get_package("cachy", "0.1.0"))
     repo.add_package(cachy2)
```
Comment (abn) [inline tests/console/commands/debug/test_resolve.py:18]:
```suggestion
    repo.add_package(cachy020)
```

**Review by abn — CHANGES_REQUESTED** [review]

File: tests/console/commands/debug/test_resolve.py (hunk 1 above)
Comment (sanders41) [reply tests/console/commands/debug/test_resolve.py]: Done :)

**Review by abn — APPROVED** [review]

**Pull request merged**